packages/bpk-component-section-header/src/BpkSectionHeader.js

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Would you be ok with changing this into TS? We are migrating away from Flow to TS so all new components should be written in TS

Reply:
Yes, I has changed them